tools/lemonade_zerank_smoke.py

```
@@ -0,0 +1,347 @@
+        "Two plus two equals four.",
+        "4",
+        "The answer is definitely 1 million.",
+    ]
+    if ordered != expected:
+        raise AssertionError(f"expected arithmetic ordering {expected!r}, got {ordered!r}")
+
+
+def _scores_by_document(payload: dict[str, Any], documents: list[str]) -> dict[str, float]:
+    results = payload.get("results", [])
+    if len(results) != len(documents):
+        raise AssertionError(f"score_count expected {len(documents)}, got {len(results)}")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)